src/components/studies/StudyBuilder.tsx

```diff
@@ -211,11 +211,11 @@ const StudyBuilder: FunctionComponent<StudyBuilderProps> = ({
       setData({
         ...builderInfo,
         schedule: savedUpdatedSchedule,
-        study: {...builderInfo.study, version: builderInfo.study.version+1} //temporary to get notifications working
+        study: { ...builderInfo.study, version: builderInfo.study.version + 1 }, //temporary to get notifications working
       })
       setHasObjectChanged(false)
     } catch (e) {
-      console.log(e)
+      console.log(e, 'error')
```

Comment:
I think this logic should happen in the scheduler component. Just set the error and move this logic in the scheduler. 
The reasons for that are that:
a. this logic scheduler specific
b. you end up parsing the errors twice: - first time here and the second time in the scheduler